presto-docs/src/main/sphinx/release/release-0.182.rst

```diff
@@ -0,0 +1,42 @@
+=============
+Release 0.182
+=============
+
+General Changes
+---------------
+
+* Add :doc:`/connector/tpcds`. This connector provides a set of schemas to
+  support the TPC Benchmark™ DS (TPC-DS).
+* Add a new :ref:`ipaddress_type` type to represent IP addresses.
+* Add a new experimental ORC writer implementation optimized for Presto.
+  We have some upcoming improvements, so we recommend waiting a few releases before
+  using this in production. The new writer can be enabled with the
+  ``hive.orc.optimized-writer.enabled`` configuration property or with the
+  ``orc_optimized_writer_enabled`` session property.
+* Add :func:`to_milliseconds` function to convert intervals (day to second) to milliseconds.
+* Add support for column aliases in ``CREATE TABLE ... AS`` statements.
+* Add a config option to reject queries during cluster initialization.
+  Queries are rejected if active worker count is less than the
+  ``query-manager.initialization-required-workers`` property while the
+  uptime of the coordinator is less than ``query-manager.initialization-timeout``.
+* Implement subtraction for all ``TIME`` and ``TIMESTAMP`` types.
+* Improve the planning performance of queries that join multiple tables with
+  a large number columns.
+* Improve the performance of joins with only non-equality conditions by using
+  a nested loops join instead of a hash join.
+* Improve the performance of ``CAST`` from ``JSON`` to ``ARRAY``/``MAP`` type.
+* Fix the :doc:`/sql/explain` query plan for tables that are partitioned
+  on timestamp/date columns.
+* Fix correctness issue that causes :func:`corr` to return positive numbers for inverse correlations.
```

Comment:
Move this to the top as correctness issues are the most important.

Then reorder everything like this: fix, improve, add

i.e., fixes, improvements / performance, new features